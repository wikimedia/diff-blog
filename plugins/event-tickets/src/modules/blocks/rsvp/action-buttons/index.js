export { default as AttendeesActionButton } from './attendees-action-button/container';
export { default as SettingsActionButton } from './settings-action-button/container';
