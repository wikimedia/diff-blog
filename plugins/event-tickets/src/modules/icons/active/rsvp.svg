<svg width="60" height="60" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M5.419 24.046l26.093 9.307 16.183-22.117.567.858-15.905 25.354a1.926 1.926 0 0 1-2.491.701L5.012 25.761l.407-1.715z" fill="#E6E6E6"/><path fill="#039ED3" d="M12.517 49.828l-1.762-.945 10.58-19.731 1.763.946zM55.309 37.557l-19.43-11.125.995-1.736 19.429 11.125z"/><path d="M3 9h54.017v41.354H3z"/><path d="M5.237 23.411l7.084 24.706L54.78 35.942l-7.084-24.706L5.237 23.411zm6.398 26.943a1.005 1.005 0 0 1-.961-.724L3.04 23.001a1 1 0 0 1 .685-1.237L48.105 9.038a1.002 1.002 0 0 1 1.237.685l7.635 26.63a1 1 0 0 1-.685 1.236l-44.38 12.726a.959.959 0 0 1-.277.04z" fill="#444"/><path d="M31.1 34.48c-.118 0-.237-.02-.352-.063L3.83 24.295l.704-1.871 26.222 9.86 17.012-22.257 1.59 1.215-17.463 22.846a1 1 0 0 1-.795.393" fill="#444"/></g></svg>