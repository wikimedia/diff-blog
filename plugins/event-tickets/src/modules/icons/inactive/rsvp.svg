<svg width="60" height="60" xmlns="http://www.w3.org/2000/svg"><g fill="#AEB4BB" fill-rule="evenodd"><path d="M12.517 49.828l-1.762-.945 10.58-19.731 1.763.946zM55.309 37.557l-19.43-11.125.995-1.736 19.429 11.125z"/><path d="M5.237 23.411l7.084 24.706L54.78 35.942l-7.084-24.706L5.237 23.411zm6.398 26.943a1.005 1.005 0 0 1-.961-.724L3.04 23.001a1 1 0 0 1 .685-1.237L48.105 9.038a1.002 1.002 0 0 1 1.237.685l7.635 26.63a1 1 0 0 1-.685 1.236l-44.38 12.726a.959.959 0 0 1-.277.04z"/><path d="M31.1 34.48c-.118 0-.237-.02-.352-.063L3.83 24.295l.704-1.871 26.222 9.86 17.012-22.257 1.59 1.215-17.463 22.846a1 1 0 0 1-.795.393"/></g></svg>