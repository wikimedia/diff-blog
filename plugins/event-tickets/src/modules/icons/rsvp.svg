<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 47.99 38.9"><defs><style>.cls-1{fill:#010101;}</style></defs><title>block-icon-rsvp</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M47.93,28l-5-26.27L26.1,27a1.11,1.11,0,0,1-1.57.3L0,10.38,5,36.29a3.21,3.21,0,0,0,3.75,2.54l36.67-7A3.21,3.21,0,0,0,47.93,28Z"/><path class="cls-1" d="M23.64,21.23a.81.81,0,0,0,1-.11L39.93,0,1.65,7.34,22.49,20.51Z"/></g></g></svg>