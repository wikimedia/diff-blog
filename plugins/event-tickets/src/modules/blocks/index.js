/**
 * Wordpress dependencies
 */
import { registerBlockType } from '@wordpress/blocks';

/**
 * Internal dependencies
 */
import { initStore } from '@moderntribe/tickets/data';
import rsvp from '@moderntribe/tickets/blocks/rsvp';
import tickets from '@moderntribe/tickets/blocks/tickets';
import ticket from '@moderntribe/tickets/blocks/ticket';
import attendees from '@moderntribe/tickets/blocks/attendees';

const blocks = [
	rsvp,
	tickets,
	ticket,
	attendees,
];

blocks.forEach( ( block ) => registerBlockType( `tribe/${ block.id }`, block ) );

// Initialize AFTER blocks are registered
// to avoid plugin shown as available in reducer
// but not having block available for use
initStore();

export default blocks;
