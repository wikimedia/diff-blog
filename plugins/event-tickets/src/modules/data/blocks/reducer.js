/**
 * External dependencies
 */
import { combineReducers } from 'redux';

/**
 * Internal dependencies
 */
import rsvp from './rsvp';
import ticket from './ticket';
import attendees from './attendees';

export default combineReducers( {
	rsvp,
	ticket,
	attendees,
} );
