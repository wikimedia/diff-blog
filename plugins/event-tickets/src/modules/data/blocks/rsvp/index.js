/**
 * Internal dependencies
 */
import * as types from './types';
import * as actions from './actions';
import * as selectors from './selectors';
import * as thunks from './thunks';
import reducer from './reducer';
import sagas from './sagas';

export default reducer;
export { types, actions, sagas, selectors, thunks };
