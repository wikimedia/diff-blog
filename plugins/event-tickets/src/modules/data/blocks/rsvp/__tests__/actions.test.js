/**
 * External dependencies
 */
import moment from 'moment';

/**
 * Internal dependencies
 */
import { actions } from '@moderntribe/tickets/data/blocks/rsvp';

describe( 'RSVP block actions', () => {
	describe( 'RSVP actions', () => {
		test( 'set RSVP id', () => {
			expect( actions.setRSVPId( 42 ) ).toMatchSnapshot();
		} );

		test( 'set RSVP settings open', () => {
			expect( actions.setRSVPSettingsOpen( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP has changes', () => {
			expect( actions.setRSVPHasChanges( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP is loading', () => {
			expect( actions.setRSVPIsLoading( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP is settings loading', () => {
			expect( actions.setRSVPIsSettingsLoading( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP is modal open', () => {
			expect( actions.setRSVPIsModalOpen( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP going count', () => {
			expect( actions.setRSVPGoingCount( 10 ) ).toMatchSnapshot();
		} );

		test( 'set RSVP not going count', () => {
			expect( actions.setRSVPNotGoingCount( 10 ) ).toMatchSnapshot();
		} );

		test( 'set RSVP has attendee info fields', () => {
			expect( actions.setRSVPHasAttendeeInfoFields( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP has duration error', () => {
			expect( actions.setRSVPHasDurationError( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP header image', () => {
			expect( actions.setRSVPHeaderImage( {
				id: 42,
				src: 'test-src',
				alt: 'test-alt',
			} ) ).toMatchSnapshot();
		} );

		test( 'set RSVP details', () => {
			expect( actions.setRSVPDetails( {
				title: 'title',
				description: 'description',
				capacity: 20,
				notGoingResponses: true,
				startDate: 'January 1, 2018',
				startDateInput: 'January 1, 2018',
				startDateMoment: moment( 'January 1, 2018' ),
				startTime: '12:34',
				endDate: 'January 4, 2018',
				endDateInput: 'January 4, 2018',
				endDateMoment: moment( 'January 4, 2018' ),
				endTime: '23:32',
				startTimeInput: '12:34',
				endTimeInput: '23:32',
			} ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp details', () => {
			expect( actions.setRSVPTempDetails( {
				tempTitle: 'title',
				tempDescription: 'description',
				tempCapacity: 20,
				tempNotGoingResponses: true,
				tempStartDate: 'January 1, 2018',
				tempStartDateInput: 'January 1, 2018',
				tempStartDateMoment: moment( 'January 1, 2018' ),
				tempStartTime: '12:34',
				tempEndDate: 'January 4, 2018',
				tempEndDateInput: 'January 4, 2018',
				tempEndDateMoment: moment( 'January 4, 2018' ),
				tempEndTime: '23:32',
				tempStartTimeInput: '12:34',
				tempEndTimeInput: '23:32',
			} ) ).toMatchSnapshot();
		} );
	} );

	describe( 'RSVP details actions', () => {
		test( 'set RSVP title', () => {
			expect( actions.setRSVPTitle( 'title' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP description', () => {
			expect( actions.setRSVPDescription( 'description' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP capacity', () => {
			expect( actions.setRSVPCapacity( 20 ) ).toMatchSnapshot();
		} );

		test( 'set RSVP not going responses', () => {
			expect( actions.setRSVPNotGoingResponses( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP start date', () => {
			expect( actions.setRSVPStartDate( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP start date input', () => {
			expect( actions.setRSVPStartDateInput( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP start date moment', () => {
			expect( actions.setRSVPStartDateMoment( moment( 'January 1, 2018' ) ) ).toMatchSnapshot();
		} );

		test( 'set RSVP end date', () => {
			expect( actions.setRSVPEndDate( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP end date input', () => {
			expect( actions.setRSVPEndDateInput( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP end date moment', () => {
			expect( actions.setRSVPEndDateMoment( moment( 'January 1, 2018' ) ) ).toMatchSnapshot();
		} );

		test( 'set RSVP start time', () => {
			expect( actions.setRSVPStartTime( '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP end time', () => {
			expect( actions.setRSVPEndTime( '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP start time input', () => {
			expect( actions.setRSVPStartTimeInput( '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP end time input', () => {
			expect( actions.setRSVPEndTimeInput( '12:34' ) ).toMatchSnapshot();
		} );
	} );

	describe( 'RSVP temp details actions', () => {
		test( 'set RSVP temp title', () => {
			expect( actions.setRSVPTempTitle( 'temp title' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp description', () => {
			expect( actions.setRSVPTempDescription( 'temp description' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp capacity', () => {
			expect( actions.setRSVPTempCapacity( 20 ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp not going responses', () => {
			expect( actions.setRSVPTempNotGoingResponses( true ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp start date', () => {
			expect( actions.setRSVPTempStartDate( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp start date input', () => {
			expect( actions.setRSVPTempStartDateInput( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp start date moment', () => {
			expect( actions.setRSVPTempStartDateMoment( moment( 'January 1, 2018' ) ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp end date', () => {
			expect( actions.setRSVPTempEndDate( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp end date input', () => {
			expect( actions.setRSVPTempEndDateInput( 'January 1, 2018' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp end date moment', () => {
			expect( actions.setRSVPTempEndDateMoment( moment( 'January 1, 2018' ) ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp start time', () => {
			expect( actions.setRSVPTempStartTime( '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp end time', () => {
			expect( actions.setRSVPTempEndTime( '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp start time input', () => {
			expect( actions.setRSVPTempStartTimeInput( '12:34' ) ).toMatchSnapshot();
		} );

		test( 'set RSVP temp end time input', () => {
			expect( actions.setRSVPTempEndTimeInput( '12:34' ) ).toMatchSnapshot();
		} );
	} );

	describe( 'RSVP thunk & saga actions', () => {
		test( 'create RSVP', () => {
			expect( actions.createRSVP() ).toMatchSnapshot();
		} );

		test( 'initialize RSVP', () => {
			expect( actions.initializeRSVP() ).toMatchSnapshot();
		} );

		test( 'delete RSVP', () => {
			expect( actions.deleteRSVP() ).toMatchSnapshot();
		} );

		test( 'handle RSVP start date', () => {
			expect( actions.handleRSVPStartDate( {} ) ).toMatchSnapshot();
		} );

		test( 'handle RSVP end date', () => {
			expect( actions.handleRSVPEndDate( {} ) ).toMatchSnapshot();
		} );

		test( 'handle RSVP start time', () => {
			expect( actions.handleRSVPStartTime( 1000 ) ).toMatchSnapshot();
		} );

		test( 'handle RSVP end time', () => {
			expect( actions.handleRSVPEndTime( 1000 ) ).toMatchSnapshot();
		} );

		test( 'fetch RSVP header image', () => {
			expect( actions.fetchRSVPHeaderImage( 10 ) ).toMatchSnapshot();
		} );

		test( 'fetch RSVP header image', () => {
			expect( actions.updateRSVPHeaderImage( {} ) ).toMatchSnapshot();
		} );

		test( 'fetch RSVP header image', () => {
			expect( actions.deleteRSVPHeaderImage() ).toMatchSnapshot();
		} );
	} );
} );
