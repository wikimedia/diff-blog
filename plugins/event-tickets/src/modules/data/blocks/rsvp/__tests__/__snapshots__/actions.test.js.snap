// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RSVP block actions RSVP actions set RSVP details 1`] = `
Object {
  "payload": Object {
    "capacity": 20,
    "description": "description",
    "endDate": "January 4, 2018",
    "endDateInput": "January 4, 2018",
    "endDateMoment": "2018-01-04T00:00:00.000Z",
    "endTime": "23:32",
    "endTimeInput": "23:32",
    "notGoingResponses": true,
    "startDate": "January 1, 2018",
    "startDateInput": "January 1, 2018",
    "startDateMoment": "2018-01-01T00:00:00.000Z",
    "startTime": "12:34",
    "startTimeInput": "12:34",
    "title": "title",
  },
  "type": "@@MT/TICKETS/SET_RSVP_DETAILS",
}
`;

exports[`RSVP block actions RSVP actions set RSVP going count 1`] = `
Object {
  "payload": Object {
    "goingCount": 10,
  },
  "type": "@@MT/TICKETS/SET_RSVP_GOING_COUNT",
}
`;

exports[`RSVP block actions RSVP actions set RSVP has attendee info fields 1`] = `
Object {
  "payload": Object {
    "hasAttendeeInfoFields": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_HAS_ATTENDEE_INFO_FIELDS",
}
`;

exports[`RSVP block actions RSVP actions set RSVP has changes 1`] = `
Object {
  "payload": Object {
    "hasChanges": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_HAS_CHANGES",
}
`;

exports[`RSVP block actions RSVP actions set RSVP has duration error 1`] = `
Object {
  "payload": Object {
    "hasDurationError": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_HAS_DURATION_ERROR",
}
`;

exports[`RSVP block actions RSVP actions set RSVP header image 1`] = `
Object {
  "payload": Object {
    "alt": "test-alt",
    "id": 42,
    "src": "test-src",
  },
  "type": "@@MT/TICKETS/SET_RSVP_HEADER_IMAGE",
}
`;

exports[`RSVP block actions RSVP actions set RSVP id 1`] = `
Object {
  "payload": Object {
    "id": 42,
  },
  "type": "@@MT/TICKETS/SET_RSVP_ID",
}
`;

exports[`RSVP block actions RSVP actions set RSVP is loading 1`] = `
Object {
  "payload": Object {
    "isLoading": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_IS_LOADING",
}
`;

exports[`RSVP block actions RSVP actions set RSVP is modal open 1`] = `
Object {
  "payload": Object {
    "isModalOpen": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_IS_MODAL_OPEN",
}
`;

exports[`RSVP block actions RSVP actions set RSVP is settings loading 1`] = `
Object {
  "payload": Object {
    "isSettingsLoading": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_IS_SETTINGS_LOADING",
}
`;

exports[`RSVP block actions RSVP actions set RSVP not going count 1`] = `
Object {
  "payload": Object {
    "notGoingCount": 10,
  },
  "type": "@@MT/TICKETS/SET_RSVP_NOT_GOING_COUNT",
}
`;

exports[`RSVP block actions RSVP actions set RSVP settings open 1`] = `
Object {
  "payload": Object {
    "settingsOpen": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_SETTINGS_OPEN",
}
`;

exports[`RSVP block actions RSVP actions set RSVP temp details 1`] = `
Object {
  "payload": Object {
    "tempCapacity": 20,
    "tempDescription": "description",
    "tempEndDate": "January 4, 2018",
    "tempEndDateInput": "January 4, 2018",
    "tempEndDateMoment": "2018-01-04T00:00:00.000Z",
    "tempEndTime": "23:32",
    "tempEndTimeInput": "23:32",
    "tempNotGoingResponses": true,
    "tempStartDate": "January 1, 2018",
    "tempStartDateInput": "January 1, 2018",
    "tempStartDateMoment": "2018-01-01T00:00:00.000Z",
    "tempStartTime": "12:34",
    "tempStartTimeInput": "12:34",
    "tempTitle": "title",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_DETAILS",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP capacity 1`] = `
Object {
  "payload": Object {
    "capacity": 20,
  },
  "type": "@@MT/TICKETS/SET_RSVP_CAPACITY",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP description 1`] = `
Object {
  "payload": Object {
    "description": "description",
  },
  "type": "@@MT/TICKETS/SET_RSVP_DESCRIPTION",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP end date 1`] = `
Object {
  "payload": Object {
    "endDate": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_END_DATE",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP end date input 1`] = `
Object {
  "payload": Object {
    "endDateInput": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_END_DATE_INPUT",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP end date moment 1`] = `
Object {
  "payload": Object {
    "endDateMoment": "2018-01-01T00:00:00.000Z",
  },
  "type": "@@MT/TICKETS/SET_RSVP_END_DATE_MOMENT",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP end time 1`] = `
Object {
  "payload": Object {
    "endTime": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_END_TIME",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP end time input 1`] = `
Object {
  "payload": Object {
    "endTimeInput": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_END_TIME_INPUT",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP not going responses 1`] = `
Object {
  "payload": Object {
    "notGoingResponses": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_NOT_GOING_RESPONSES",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP start date 1`] = `
Object {
  "payload": Object {
    "startDate": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_START_DATE",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP start date input 1`] = `
Object {
  "payload": Object {
    "startDateInput": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_START_DATE_INPUT",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP start date moment 1`] = `
Object {
  "payload": Object {
    "startDateMoment": "2018-01-01T00:00:00.000Z",
  },
  "type": "@@MT/TICKETS/SET_RSVP_START_DATE_MOMENT",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP start time 1`] = `
Object {
  "payload": Object {
    "startTime": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_START_TIME",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP start time input 1`] = `
Object {
  "payload": Object {
    "startTimeInput": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_START_TIME_INPUT",
}
`;

exports[`RSVP block actions RSVP details actions set RSVP title 1`] = `
Object {
  "payload": Object {
    "title": "title",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TITLE",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp capacity 1`] = `
Object {
  "payload": Object {
    "capacity": 20,
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_CAPACITY",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp description 1`] = `
Object {
  "payload": Object {
    "description": "temp description",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_DESCRIPTION",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp end date 1`] = `
Object {
  "payload": Object {
    "endDate": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_END_DATE",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp end date input 1`] = `
Object {
  "payload": Object {
    "endDateInput": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_END_DATE_INPUT",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp end date moment 1`] = `
Object {
  "payload": Object {
    "endDateMoment": "2018-01-01T00:00:00.000Z",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_END_DATE_MOMENT",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp end time 1`] = `
Object {
  "payload": Object {
    "endTime": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_END_TIME",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp end time input 1`] = `
Object {
  "payload": Object {
    "endTimeInput": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_END_TIME_INPUT",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp not going responses 1`] = `
Object {
  "payload": Object {
    "notGoingResponses": true,
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_NOT_GOING_RESPONSES",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp start date 1`] = `
Object {
  "payload": Object {
    "startDate": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_START_DATE",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp start date input 1`] = `
Object {
  "payload": Object {
    "startDateInput": "January 1, 2018",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_START_DATE_INPUT",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp start date moment 1`] = `
Object {
  "payload": Object {
    "startDateMoment": "2018-01-01T00:00:00.000Z",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_START_DATE_MOMENT",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp start time 1`] = `
Object {
  "payload": Object {
    "startTime": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_START_TIME",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp start time input 1`] = `
Object {
  "payload": Object {
    "startTimeInput": "12:34",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_START_TIME_INPUT",
}
`;

exports[`RSVP block actions RSVP temp details actions set RSVP temp title 1`] = `
Object {
  "payload": Object {
    "title": "temp title",
  },
  "type": "@@MT/TICKETS/SET_RSVP_TEMP_TITLE",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions create RSVP 1`] = `
Object {
  "type": "@@MT/TICKETS/CREATE_RSVP",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions delete RSVP 1`] = `
Object {
  "type": "@@MT/TICKETS/DELETE_RSVP",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions fetch RSVP header image 1`] = `
Object {
  "payload": Object {
    "id": 10,
  },
  "type": "@@MT/TICKETS/FETCH_RSVP_HEADER_IMAGE",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions fetch RSVP header image 2`] = `
Object {
  "payload": Object {
    "image": Object {},
  },
  "type": "@@MT/TICKETS/UPDATE_RSVP_HEADER_IMAGE",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions fetch RSVP header image 3`] = `
Object {
  "type": "@@MT/TICKETS/DELETE_RSVP_HEADER_IMAGE",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions handle RSVP end date 1`] = `
Object {
  "payload": Object {},
  "type": "@@MT/TICKETS/HANDLE_RSVP_END_DATE",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions handle RSVP end time 1`] = `
Object {
  "payload": Object {
    "seconds": 1000,
  },
  "type": "@@MT/TICKETS/HANDLE_RSVP_END_TIME",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions handle RSVP start date 1`] = `
Object {
  "payload": Object {},
  "type": "@@MT/TICKETS/HANDLE_RSVP_START_DATE",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions handle RSVP start time 1`] = `
Object {
  "payload": Object {
    "seconds": 1000,
  },
  "type": "@@MT/TICKETS/HANDLE_RSVP_START_TIME",
}
`;

exports[`RSVP block actions RSVP thunk & saga actions initialize RSVP 1`] = `
Object {
  "type": "@@MT/TICKETS/INITIALIZE_RSVP",
}
`;
